### PR TITLE
Wavelet transform analysis of images using WaveletAnalyzer toolbox in Matlab

### LINKED ISSUE #6655: Wavelet transform analysis of images using WaveletAnalyzer toolbox in Matlab
### Topic Suggestion
Wavelet transform analysis of images using WaveletAnalyzer toolbox in Matlab

### Writing sample(s): 
https://www.section.io/engineering-education/wavelet-transform-based-denoising-of-images-using-matlab/

#### Introduction
The wavelet transform is signal decomposition using a system of wavelets, that is, functions, each of which is a shifted and scaled copy of a function, the mother wavelet. The term wavelet transform defines a class of decomposition methods. Wavelet Analyzer app is an interactive tool for using wavelets to visualize and analyze signals and images. This App can be used for applications such as wavelet transform analysis, denoising images and signals, e.t.c.
This article covers wavelet analysis and synthesis of images using filter banks. Also, it shows how to perform wavelet transform analysis of images using the WaveletAnalyzer toolbox. How to use the various wavelet functions such as dwt2, idwt2, wavedec2 and waverec2 are covered here.

#### Key takeaways 
- Wavelet analysis and synthesis of images using filter banks.
- Wavelet transform analysis of images using WaveletAnalyzer toolbox of Matlab.
- Use of dwt2, idwt2, wavedec2 and waverec2 Matlab function.

#### Article quality
This article shows how to use the WaveletAnalyzer toolbox and the various Matlab functions.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by atienodorine3**
## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #6655 

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by ahmadmardeni1** [comment]
Plagiarism check done ✅

File: content/articles/wavelet-transform-analysis-of-images-using-waveletanalyzer-toolbox-in-matlab/index.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+author: atieno-dorine
+date: 2022-03-18T00:00:00-02:25
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/wavelet-transform-analysis-of-images-using-waveletanalyzer-toolbox-in-matlab/hero.jpg
+    alt: Wavelet transform analysis of images using WaveletAnalyzer toolbox in Matlab Hero Image
+---
+The wavelet transform is the decomposition of signals using wavelet techniques. This function creates functions that are then shifted and a copy of the mother wavelet.
```
Comment (M0nicah) [inline content/articles/wavelet-transform-analysis-of-images-using-waveletanalyzer-toolbox-in-matlab/index.md:17]: this sentence appears to be missing something .🤔

```
This function creates functions that are then shifted and a copy of the mother wavelet.
```

**Review by M0nicah — CHANGES_REQUESTED** [review]
hi @atienodorine3, kindly take a look at the sentence

**Comment by atienodorine3** [comment]
@M0nicah I have corrected the sentence. Thank you.

**Review by M0nicah — APPROVED** [review]
great article @atienodorine3, ensure that you go through the changes I made in the commits to improve flow and format of your future articles. 🚀

**Pull request merged**